app/views/outcomes-report/OutcomesReportResultComponent.vue

```diff
@@ -236,6 +236,24 @@ export default Vue.extend({
 
     isSchoolAdmin () {
       return me.isSchoolAdmin()
+    },
+
+    topTeacherInfo () {
+      const formatTeacher = (tea) => {
+        if (!tea) {
+          return ''
+        }
+        return `<a href="/outcomes-report/teacher/${tea._id}">${tea.firstName}</a>`
+      }
+      const info = $.i18n.t('outcomes.top_teacher_info', {
+        A: formatTeacher(this.org.topTeachers[0]),
+        B: formatTeacher(this.org.topTeachers[1]),
+        n: this.org.nces.teachers - 2
+      })
+      // $.i18.t encode the html
+      const txt = document.createElement('textarea')
+      txt.innerHTML = info
```

Comment:
do we need to do it like this? Why not pass values as variables to vue template instead? Could also just create a component which accepts these parameters

Reply:
ya we can use component too. but component may complex than a single document element?